### PR TITLE
Adds 'custom' HPC option to React workflow.

### PR DESCRIPTION
**Pull request by GabrielBram**
Some of the solutions are a bit hacky here, but I am working on the assumption that only users with some experience of setting up their AIMs commands would be interested in this. This allows flexibility to set custom FHI-aims commands, meaning React can be run on desktops/alternative compilations without changing hardcoded values.

Needing a new environmental variable for the FHI-aims root directory is also less than desirable, but intended to avoid: A) expecting a key word on the React level, which must be passed down all the way to AIMs path, B) problems with the AIMS_SPECIES_DIRECTORY, which would not work with the existing logic for selecting basis levels.

### CODE REVIEW

File: carmm/run/workflows/react.py
Comment (ikowalec) [inline carmm/run/workflows/react.py:1]: I'm not a fan of this massive print out, perhaps a proper logger should be used at a `warning` level? Food for thought, other instances of verbosity in the output would need to be adjusted also.

**Comment by ikowalec** [comment]
Nothing wrong with this, but given that the way the handling of the FHI-aims species directory and command of `Aims` calculator are handled in ASE `3.23`, we might want to think about future-proofing the method. My recommendation is to leave that as a TODO prior to merge.

**Review by ikowalec — APPROVED** [review]
This in principle should work well, although a bit hacky. The way the FHI-aims environmental variables are set up is going to change in 3.23 anyways, so will only be a temporary solution. Also add the "custom" `hpc `setting to the React regression test to increase coverage.

File: carmm/run/workflows/react.py
Comment (GabrielBram) [reply carmm/run/workflows/react.py:1]: It is a little verbose - will add some warning supression.

**Pull request merged**